### PR TITLE
Add VLO

### PR DESCRIPTION
**Pull request by Velosmo-1**
Add $VLO token

### CODE REVIEW

File: juno/assetlist.json [hunk 1]
```diff
@@ -1174,6 +1174,28 @@
...
+        {
+          "denom": "vlo",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno1ezdrrmzk3d5ws86ux2wxuf7q5plgyp0upjryh4zcm96ufp7rp05qt2r2gu",
+      "name": "Velosmo",
+      "display": "vlo",
+      "symbol": "VLO",
+      "logo_URIs": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/velosmo.png"
```
Comment (Velosmo-1) [inline juno/assetlist.json:1197]: https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/velosmo.svg

File: juno/assetlist.json [hunk 2]
```diff
@@ -1174,6 +1174,28 @@
...
+        "png": {
```
[the remark this answers is not in the document]
Comment (JeremyParish69) [reply juno/assetlist.json:1198]: I see a separate PR for the svg image. Could we please add all the images into the same PR?

[the remark this answers is not in the document]
Comment (Velosmo-1) [reply juno/assetlist.json:1198]: Hi, can you watch on discord I sent you a message ?
MVelosmo#2991

**Comment by JeremyParish69** [comment]
Assuming this effort has been abandoned. Can be reopened if needed

**Pull request closed without merging**